railties/test/abstract_unit.rb

```diff
@@ -16,7 +16,7 @@
 
 module TestApp
   class Application < Rails::Application
-    config.root = __dir__
+    config.root = File.expand_path("../../", __FILE__)
```

Comment:
I think this may have been the root cause of the issue?

Reply:
I was not expecting that to be wrong, but now you mention it..! 👍🏻

Reply:
@byroot Looks like that path has been used for 18 years though, so I'm wondering how wrong that could have been... 🤔 
https://github.com/davidstosik/rails/blob/4e3ed5bc44f6cd20c9e353ab63fd24b92a7942be/railties/test/abstract_unit.rb#L9-L13

Reply:
Yeah, but `bin/rails test` is much more recent, so it probably was never a problem before.